src/qt/overviewpage.cpp

```diff
@@ -87,15 +76,6 @@ class TxViewDelegate : public QAbstractItemDelegate
             iconWatchonly.paint(painter, watchonlyRect);
         }
 
-        if(fConflicted) { // No need to check anything else for conflicted transactions
-            foreground = COLOR_CONFLICTED;
-        } else if (!confirmed || fImmature) {
-            foreground = COLOR_UNCONFIRMED;
-        } else if (amount < 0) {
-            foreground = COLOR_NEGATIVE;
-        } else {
-            foreground = COLOR_BLACK;
-        }
```

Comment:
Would prefer to add some extra transaction type checking rather than remove these status checks here.

Reply:
Transaction type checking is already done in the transaction table model.
Conflicted (non orphan stakes) transaction are already in red (`COLOR_CONFLICTED`).
I would prefer the amount to be printed with the same color as the address (except for negative amounts).